modules/installation-guide/partials/proc_importing-untrusted-tls-certificates.adoc

```diff
@@ -22,121 +25,90 @@ endif::[]
 ifeval::["{project-context}" != "che"]
 When
 endif::[]
-the cluster contains cluster-wide trusted CA certificates added through the link:https://docs.openshift.com/container-platform/4.4/networking/configuring-a-custom-pki.html#nw-proxy-configure-object_configuring-a-custom-pki[cluster-wide-proxy configuration], {prod-short} Operator detects them and automatically injects them into this ConfigMap:
+the cluster contains cluster-wide trusted CA certificates added through the link:https://docs.openshift.com/container-platform/4.4/networking/configuring-a-custom-pki.html#nw-proxy-configure-object_configuring-a-custom-pki[cluster-wide-proxy configuration], {prod-short} Operator detects them and automatically injects them into this config map:
 
-- {prod-short} automatically labels the ConfigMap with the `config.openshift.io/inject-trusted-cabundle="true"` label.
-- Based on this annotation, OpenShift automatically injects the cluster-wide trusted CA certificates inside the `ca-bundle.crt` key of ConfigMap
+- {prod-short} automatically labels the config map with the `config.openshift.io/inject-trusted-cabundle="true"` label.
+- Based on this annotation, OpenShift automatically injects the cluster-wide trusted CA certificates inside the `ca-bundle.crt` key of config map
 ====
 
-== At {prod-short} installation time
+== Adding new CA certificates into {prod-short}
+
+This guide can be used before the installations of {prod-short} or when {prod-short} is already installed and running.
+
+include::example$snip_importing-untrusted-tls-certificates_1.adoc[]
 
 .Prerequisites
 
 * The `{orch-cli}` tool is available.
-* You are ready to create `CheCluster` custom resource.
+* Namespace for {prod-short} exists.
 
 .Procedure
 
-include::example$snip_importing-untrusted-tls-certificates_1.adoc[]
-
-. Create a new ConfigMap with the required TLS certificates:
+. Create a new config map with the required TLS certificates:
 +
 [subs="+attributes,+quotes"]
 ----
-$ {orch-cli} create configmap ca-certs --from-file=__<certificate-file-path>__ -n=__<{project-context}-namespace-name>__
+$ {orch-cli} create configmap custom-certs --from-file=__<bundle-file-path>__ -n=__<{project-context}-namespace-name>__
 ----
 +
-To apply more than one certificate, add another `--from-file=_<certificate-file-path>_` option to the above command.
+To apply more than one bundle, add another `--from-file=_<bundle-file-path>_` option to the above command.
+Or you may create another config map.
 
-. During the installation process, when creating the `CheCluster` custom resource, take care of configuring the right name for the created ConfigMap.
```

Comment:
I believe we should keep this section regarding `spec.server.ServerTrustStoreConfigMapName` but mention that it is relevant for the specific che version